app/views/candidate_interface/equality_and_diversity/start.html.erb

```diff
@@ -1,12 +1,12 @@
 <% content_for :title, t('equality_and_diversity.title') %>
```

Comment:
This entire page should be removed. So when a candidate selects the link 'Equality and diversity questions' link from the task list, the first question they see should be the one about 'Sex'.